apps/api/src/app/core/core.module.ts

```diff
@@ -20,6 +21,7 @@ import { MarketDataService } from './market-data.service';
     CurrentRateService,
     DataProviderService,
     ExchangeRateDataService,
```

Comment:
Do you have the goal to eliminate this service?

Reply:
I integrated the `ExchangeRateService` inside the `ExchangeRateDataService`. Both methods are co-existing now and I think this is totally fine. There are usages of the current `toCurrency` method which are typically using the current market price. While the new methods are used for the historic data.